linchpin/__init__.py

```diff
@@ -132,7 +132,7 @@ def up(ctx, pinfile, targets):
 
     \b
     TARGET ...
-    Provision ONLY the listed target(s).
+    Provision ONLY the listed target(s). If omitted, ALL targets will be provisioned.
```

Comment:
Great idea here! Maybe it should read 'If omitted, ALL targets in the appropriate PinFile will be provisioned.'